README.md

```diff
@@ -37,6 +37,71 @@ Latest API Docs is <a target="_blank" href="https://wso2-extensions.github.io/si
      </dependency>
 ```
 
+## Running Integration tests in docker containers(Optional)
+
+1. Install and run docker in daemon mode.
+
+    *  How to install docker on Linux
+    ```    
+    wget -qO- https://get.docker.com/ | sh
+    sudo service dockerd stop
+    export DOCKER_HOST=tcp://172.17.0.1:4326
+    docker daemon -H tcp://172.17.0.1:4326
+    ```
+    *  How to install docker on Mac 
+   https://docs.docker.com/docker-for-mac/
```

Comment:
When asking the user to issue a command, specify the purpose and the location from which it is issued.

Reply:
There is no specific location for issues this command, these are the bash script commands which use for install the docker. Yes, we can specify the purpose.

Reply:
Fixed by commit: a914e9f5a11a54dcce4b6459d5bc0c1e3e31da42